### PR TITLE
[EDU-765] CHORES Jwt authentication nodejs fix

### PR DESCRIPTION
**Pull request by fliptopbox**
Please see complimentary changes to tutorial (https://github.com/ably/docs/pull/1502)

- replaced old logo with static asset
- replaced old UI with ably-ui like form
- updated JavaScript to ES6
- Added missing `package.json`
- oh yeah and gave the demo CSS a slice of :heart: it really needed it.

2022-07-28-130106_1254x1385_scrot

# Related work:
- Update the live demo CSS for docs on `Website` page [Website PR](https://github.com/ably/website/pull/4736)
- Update the `docs` tutorial, to use the update screenshot and updated JavaScript  [docs tutorial PR](https://github.com/ably/tutorials/pull/104)
- Update the tutorials `source code`, and replace out-of-date ably logo. [Source code PR](https://github.com/ably/docs/pull/1502)

### CODE REVIEW

**Comment by fliptopbox** [comment]
@marklewin for consistency i added you to review, since you have context of the textile file, that links to this repo.

File: index.html [hunk 1]
```diff
@@ -1,39 +1,62 @@
...
+        // Send user details to the auth server
+        const authUrl = `/auth?clientId=${username}:${password}`;
+        const realtime = new Ably.Realtime({ authUrl });
+
+        console.log("Fetching JWT token from auth server", username, password);
+
+        realtime.connection.once("connected", () => {
+          const { tokenDetails } = realtime.auth;
+          console.log("Client connected to Ably using JWT", tokenDetails);
+          alert("Client successfully connected Ably using JWT auth");
```
Comment (marklewin) [inline index.html]:
```suggestion
          alert("Client successfully connected to Ably using JWT auth");
```

**Review by marklewin — CHANGES_REQUESTED** [review]
Much better! Small typo in the UI though.

**Review by marklewin — APPROVED** [review]
LGTM

**Pull request merged**